src/main/java/com/github/checkstyle/regression/git/GitChange.java

```diff
@@ -0,0 +1,47 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2017 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.github.checkstyle.regression.git;
+
+import org.eclipse.jgit.diff.DiffEntry;
```

Comment:
This may be another issue we can do once I see how this class will be used.

------

If we are passing this class to other components, than `org.eclipse.jgit` can't be in this class and this class must go to `com.github.checkstyle.regression.data` as I defined in #2 .

I don't want jgit showing up in import controls of other components as it makes it look like they are doing things outside of their purview.
I think we should expand the fields to what we need, the file path and the lines that have changed.